cubeseed_login/src/component/tables/CertificateTable.tsx

```diff
@@ -0,0 +1,309 @@
+    {
+      id: 10,
+      name: "farmer10",
+      email: "farmer10@gmail.com",
+      time: "5/1/23",
+      certificate: "document.pdf",
+      code: null,
+      document1: "Utility Bill",
+      document2: "Picture of Farm",
+    },
+    {
+      id: 11,
+      name: "farmer11",
+      email: "farmer11@gmail.com",
+      time: "5/1/23",
+      certificate: "document.pdf",
+      code: null,
+      document1: "Picture of Farm",
+      document2: "Document of Land Ownership",
+    },
+    {
+      id: 12,
+      name: "farmer12",
+      email: "farmer12@gmail.com",
+      time: "5/1/23",
+      certificate: null,
+      code: 1234,
+      document1: "Document of Land Ownership",
+      document2: "Utility Bill",
+    },
+    {
+      id: 13,
+      name: "farmer13",
+      email: "farmer13@gmail.com",
+      time: "5/1/23",
+      certificate: null,
+      code: 1234,
+      document1: "Utility Bill",
+      document2: "Picture of Farm",
+    },
+    {
+      id: 14,
+      name: "farmer14",
+      email: "farmer14@gmail.com",
+      time: "5/1/23",
+      certificate: "document.pdf",
+      code: null,
+      document1: "Picture of Farm",
+      document2: "Document of Land Ownership",
+    },
+    {
+      id: 15,
+      name: "farmer15",
+      email: "farmer15@gmail.com",
+      time: "5/1/23",
+      certificate: null,
+      code: 1234,
+      document1: "Document of Land Ownership",
+      document2: "Utility Bill",
+    },
+  ]
+
+  const [revealButtons, setRevealButtons] = useState(false)
+  const [selectedFarmer, setSelectedFarmer] = useState<Farmer | null>(null)
+  const [farmers, setFarmers] = useState([])
+  const [currentPage, setCurrentPage] = useState(0)
+  const itemsPerPage = 10
+  const startIndex = currentPage * itemsPerPage
+  const endIndex = startIndex + itemsPerPage
+  const displayedFarmers = sampleFarmers.slice(startIndex, endIndex)
+
+  /* ---uncomment useEffect when api endpoint is added--- */
+
+  // useEffect(() => {
+  //   const getFarmers = async () => {
+  //     try {
+  //       const response = await fetch(/* api endpoint here */);
+  //       const data: Farmer[] = await response.json();
+  //       setFarmers(data);
+  //     } catch (error) {
+  //       console.error(error);
+  //     }
+  //   };
+  //   getFarmers();
+  // }, [farmers]);
+
+  const approveDocument = async (farmer: Farmer) => {
+    try {
+      // api endpoint or function passed from parent component here //
+      // functionality for approval email here //
+    } catch (error) {
+      console.log(error)
+    }
+  }
+
+  const rejectDocument = async (farmer: Farmer) => {
+    try {
+      // api call or function passed from parent component here //
+      // functionality for rejection email here //
+    } catch (error) {
+      console.log(error)
+    }
+  }
+
+  const handlePageClick = (selectedPage: { selected: number }) => {
+    setCurrentPage(selectedPage.selected)
+  }
+
+  const handleButtons = (farmer: Farmer) => {
```

Comment:
and the `handleButtons `could be a custom function or hook that can be used both on newfarmerstable and here